packages/js-drive/lib/abci/handlers/prepareProposalHandlerFactory.js

```diff
@@ -42,9 +103,51 @@ function prepareProposalHandlerFactory() {
         tx,
         action: txAction.UNMODIFIED,
       });
+
+      const {
+        txResult,
+        actualProcessingFee,
+        actualStorageFee,
+      } = await deliverTx(tx, round, consensusLogger);
+
+      if (txResult.code === 0) {
+        validTxCount += 1;
+        storageFee += actualStorageFee;
+        processingFee += actualProcessingFee;
```

Comment:
Please add todo: We probably should calculate fees for invalid transitions as well